core/interop/pinvoke/marshalling/src/MarshallingSample/MarshallingSampleNative.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace MarshallingSample
+{
+    internal static class MarshallingSampleNative
+    {
+        // String marshalling APIs
+        [DllImport(nameof(MarshallingSampleNative), ExactSpelling = true, CharSet = CharSet.Ansi)]
```

Comment:
I would remove the `ExactSpelling = true` if possible.

Reply:
I followed the recommendation here: https://docs.microsoft.com/en-us/dotnet/standard/native-interop/best-practices#dllimport-attribute-settings
It says that the recommended value is true (which is not the default) because it is slightly faster.

I'm fine removing it, but if it's our recommendation...

Reply:
Boo. Okay. Feel free to leave it. Thanks for keeping us honest.